<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>906</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>906</width>
    <height>661</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FaceLock</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>921</width>
      <height>411</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Enroll_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>580</y>
      <width>83</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enroll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>83</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>580</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>410</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>580</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resoultion</string>
    </property>
   </widget>
   <widget class="QComboBox" name="AllCam">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>580</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="identify">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>580</y>
      <width>83</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Identify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>580</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>580</y>
      <width>83</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>Jobs</string>
    </property>
    <addaction name="Identify"/>
    <addaction name="Enroll"/>
    <addaction name="actionDelete"/>
    <addaction name="Cancel"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuDevices">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionVideo_Setting"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDevices_2">
    <property name="title">
     <string>Devices</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuDevices"/>
   <addaction name="menuHelp"/>
   <addaction name="menuDevices_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Enroll">
   <property name="text">
    <string>Enroll</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="Identify">
   <property name="text">
    <string>Identify</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="Setting">
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="Cancel">
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionVideo_Setting">
   <property name="text">
    <string>Video-Settings</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
